spec/bolt/executor_spec.rb

```diff
@@ -335,33 +406,6 @@ def mock_node_results
     end
   end
 
-  context 'prompting' do
```

Comment:
Where did these go?

Reply:
Ugh. That's probably VS Code. It's been having weird caching issues the past few weeks when I switch branches.